frontend/src/components/Header.vue

```diff
@@ -0,0 +1,46 @@
+<template lang="pug">
+    div
+        v-app-bar(app clipped-right color="primary" )
+            v-app-bar-nav-icon.hidden-md-and-up(dark @click.stop="drawer = !drawer")
+            v-avatar(size="48px")
+                v-img(src="../assets/logo.jpeg")
+            v-toolbar-title.hidden-sm-and-down.white--text
+                span.title.ml-3.mr-5 Devlup Labs&nbsp;
+            v-spacer
+            div.hidden-sm-and-down.my-2
+              v-btn(v-for="link in links" :key="link.text" :to="{name:link.text}" rounded depressed color="primary" exact) {{link.text}}
+              v-btn(rounded depressed color="primary" @click="() => this.$vuetify.goTo('#bottom')") {{contact.text}}
```

Comment:
Contacts will be separate page right @anshulahuja98 ? We do not need to got to footer I think?

Reply:
Yep

Reply:
This way we can avoid requesting for another page

Reply:
@vineethvanga18 Add the contact button with the test on the buttons.

Reply:
Change contacts option to Teams

Reply:
and don't redirect it to bottom

Reply:
Should I remove contacts from footer

Reply:
Footer probably just has the social links right?
Let them be there

Or did you include other stuff as well?

Reply:
only social links

Reply:
Let them be there then

Reply:
So there are the changes needed?
Change contacts option to Team
and don't redirect it to bottom

Reply:
*these

Reply:
Yes, you can include this button with the above button.